armotypes/notificationreporttypes.go

```diff
@@ -0,0 +1,151 @@
+	ImgTag             string                         `json:"imageTag" bson:"imageTag"`
+	RelatedPackageName string                         `json:"packageName" bson:"packageName"`
+	PackageVersion     string                         `json:"packageVersion" bson:"packageVersion"`
+	Link               string                         `json:"link" bson:"link"`
+	Description        string                         `json:"description" bson:"description"`
+	Severity           string                         `json:"severity" bson:"severity"`
+	SeverityScore      int                            `json:"severityScore" bson:"severityScore"`
+	Metadata           interface{}                    `json:"metadata" bson:"metadata"`
+	Fixes              VulFixes                       `json:"fixedIn" bson:"fixedIn"`
+	Relevancy          string                         `json:"relevant" bson:"relevant"`
+	UrgentCount        int                            `json:"urgent" bson:"urgent"`
+	NeglectedCount     int                            `json:"neglected" bson:"neglected"`
+	HealthStatus       string                         `json:"healthStatus" bson:"healthStatus"`
+	Categories         VulnerabilityCategory          `json:"categories" bson:"categories"`
+	ExceptionApplied   []VulnerabilityExceptionPolicy `json:"exceptionApplied,omitempty" bson:"exceptionApplied,omitempty"`
+}
+
+type VulFixes []FixedIn
+
+type VulnerabilityCategory struct {
+	IsRCE bool `json:"isRce" bson:"isRce"`
+}
+
+type FixedIn struct {
+	Name    string `json:"name" bson:"name"`
+	ImgTag  string `json:"imageTag" bson:"imageTag"`
+	Version string `json:"version" bson:"version"`
+}
+type TopVulItem struct {
+	Vulnerability   `json:",inline"`
+	WorkloadsCount  int64 `json:"workloadsCount" bson:"workloadsCount"`
+	SeverityOverall int64 `json:"severityOverall" bson:"severityOverall"`
+}
+
+type TopCtrlItem struct {
+	ControlID            string           `json:"id" bson:"id"`
+	ControlGUID          string           `json:"guid" bson:"guid"`
+	Name                 string           `json:"name" bson:"name"`
+	Remediation          string           `json:"remediation" bson:"remediation"`
+	Description          string           `json:"description" bson:"description"`
+	ClustersCount        int64            `json:"clustersCount" bson:"clustersCount"`
+	SeverityOverall      int64            `json:"severityOverall" bson:"severityOverall"`
+	BaseScore            int64            `json:"baseScore" bson:"baseScore"`
+	Clusters             []TopCtrlCluster `json:"clusters" bson:"clusters"`
+	TotalFailedResources int64            `json:"-"`
+}
+
+type TopCtrlCluster struct {
+	Name               string `json:"name" bson:"name"`
+	ResourcesCount     int64  `json:"resourcesCount" bson:"resourcesCount"`
+	ReportGUID         string `json:"reportGUID" bson:"reportGUID"`
+	TopFailedFramework string `json:"topFailedFramework" bson:"topFailedFramework"`
+}
+
+type CommonContainerScanSummaryResult struct {
```

Comment:
Is this related to notifications ?

Reply:
Let's move the container/posture scan structs to dedicated files